src/routes/outcomes/tree.ts

```diff
@@ -85,35 +88,24 @@ export const tree: TreeNodeType = {
 						"While it is unlikely that the AI cares about humans at all, there's a good chance that if it cares about humans, it also cares about our well-being.",
 					yes: {
 						type: 'safe',
-						text: 'The AI has taken control away from humans, but still deeply cares about human well-being. A weird utopia, but a utopia nonetheless.'
+						text: 'The AI has taken control away from humans, but still deeply cares about human well-being. A weird utopia, but a utopia nonetheless.' + optimist_help_str
 					},
 					no: {
 						type: 'doom',
-						text: "We're ending up with an AI that does want to keep humans alive, but does not want make us happy. We can be used as slaves, to experiment on, or even to torture. Forever."
+						text: "We're ending up with an AI that does want to keep humans alive, but does not want make us happy. We can be used as slaves, to experiment on, or even to torture. Forever." + doomer_hope_str
 					}
 				},
 				no: {
 					type: 'doom',
-					text: 'At some point, an AI will probably succeed in taking over control from humans. AIs have many evolutionary advantages over humans. We will be unable to stop it. The AI will become the dominant organism on earth, and will eventually outcompete all lifeforms on earth. Nothing survives.'
+					text: 'At some point, an AI will probably succeed in taking over control from humans. AIs have many evolutionary advantages over humans. We will be unable to stop it. The AI will become the dominant organism on earth, and will eventually outcompete all lifeforms on earth. Nothing survives.' + doomer_hope_str
 				}
 			},
 			no: humanTakeOver
 		},
 		no: humanTakeOver
 	},
 	no: {
-		type: 'question',
-		text: 'Will non-superhuman AI cause a global catastrophe?',
-		probability: 0.5,
-		explanation:
-			"AI-powered cybersecurity, autonomous weapons and engineered pandemics or bioweapons can pose large scale threats to humanity, both from misuse or accidents. We may be able to stop these threats, if we apply strict regulations and safety measures, but as of now we're not doing that.",
-		yes: {
-			type: 'doom',
-			text: 'An AI-related catastrophe happens before we even get to superintelligence. Since some dangerous capabilities do not even require general superintelligence, this might be one of the more likely outcomes.'
-		},
-		no: {
-			type: 'safe',
-			text: "If we successfully prevent dangerous AI from being developed, it can never outsmart humans, and we can even prevent other forms of AI catastrophe. We hope we're living in this universe. You can help to achieve this!"
-		}
+		type: 'safe',
+		text: "If we successfully prevent dangerous AI from being developed we can prevent AI catastrophes. We hope we're living in this universe. You can help achieve this!"
```

Comment:
Yeah this is probably more consistent with our proposal / ask. But it may feel a bit disingenuous - I do think we're still at risk of disaster even if we pause!

Reply:
sure, it was like that. I changed the can for could now.